tests/Functional/Fixtures/TestKernel.php

```diff
@@ -20,7 +20,7 @@ public function __construct(string $testCaseFile, string $commonFile = 'common.y
         $this->commonFile = $commonFile;
     }
 
-    public function registerBundles()
+    public function registerBundles(): iterable
```

Comment:
Let's use `array` as a return type.

The reason for this is that [`iterable`](https://www.php.net/manual/en/language.types.iterable.php) was introduced in PHP 7.1.0 and we state that our package works with `^7.0`.

Since the actual return type for `KernelInterface::KernelInterface()` was [added in Symfony 6.0.0](https://github.com/symfony/symfony/blob/v6.0.0/src/Symfony/Component/HttpKernel/KernelInterface.php#L32) (which requires PHP `>=8.0.2`) and return types in PHP are fully [covariant since PHP 7.4](https://www.php.net/manual/en/language.oop5.variance.php) we can use `array` and it should work for all versions of PHP and Symfony.